test/writer/drift_model_writer/primary_key_single/config.txt

```diff
@@ -0,0 +1,9 @@
+Person:
+  path: user/person/
+  generate_drift_table: true
+  properties:
+    firstName:
+      type: string
+      required: true
+      is_table_primary_key: true
+    lastName: string
```

Comment:
output.txt is generating a nullable table? this is not correct.

Reply:
We need to fix #126, I'll do that first 🙈

Reply:
Fixed in #128